### PR TITLE
Update define-mail-flow-rules-to-encrypt-email.md

### PR DESCRIPTION
**Pull request by samschan-ms**
add a comment that we reject mail if mail flow rule cannot encrypt mail.

### CODE REVIEW

**Comment by samschan-ms** [comment]
@KCCross to review/approve

File: microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ description: "Admins can learn to create mail flow rules (transport rules) to en
 
 As an administrator that manages Exchange Online, you can create mail flow rules (also known as transport rules) to help protect email messages you send and receive. You can set up rules to encrypt any outgoing email messages and remove encryption from encrypted messages coming from inside your organization or from replies to encrypted messages sent from your organization. You can use the <a href="https://go.microsoft.com/fwlink/p/?linkid=2059104" target="_blank">Exchange admin center (EAC)</a> or Exchange Online PowerShell to create these rules. In addition to overall encryption rules, you can also choose to enable or disable individual message encryption options for end users.
 
-You can't encrypt inbound mail from senders outside of your organization.
+You can't encrypt inbound mail from senders outside of your organization.  If a mail flow rule is setup to encrypt mail that Exchange Online considers coming from outside the organization, mail will be rejected and returned to the sender.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/define-mail-flow-rules-to-encrypt-email.md]:
```suggestion
You can't encrypt inbound mail from senders outside of your organization. If a mail flow rule is set up to encrypt mail that Exchange Online considers coming from outside the organization, mail will be rejected and returned to the sender.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @samschan-ms, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by samschan-ms** [comment]
@KCCross there is an ask to specify the NDR message into this change. I don't think we should put the specific on this. If we do require it, I want your expertise on where to place this.

**Pull request merged**